src/utils/testfiles/three_search_results.json

```diff
@@ -1,24 +1,22 @@
 [
```

Comment:
for min del kan vi godt gjøre om alle disse mock-filene til `.ts` og bruke typer, så blir det enklere å holde de i sync. men må ikke gjøre det nå

Reply:
godt poeng! Kan gjøre det en dag vi ikke har noe bedre å gjøre 😅

Reply:
gjorde det nå jeg - ble så mye frem og tilbake på typene 😅